### PR TITLE
Basic React Router installation

### PR DESCRIPTION
**Pull request by harcod**
There is one route "/about" that shows a prototype for an About Us page.  There is no direct navigation to this page in this PR.

"/" shows the current site <App />
"/about" shows the prototype for an About Us page. <About />
every other route redirects to "/" (otherwise we would get a 404 error).

### CODE REVIEW

**Comment by harcod** [comment]
I fixed the "color contrast" Lighthouse error, but now the rel-preconnect fails again.

> uses-rel-preconnect` failure for `maxLength` assertion (Preconnect to required origins: https://web.dev/uses-rel-preconnect/ )

File: src/About.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+                >
+                    making a contribution
+                </a>
+                . If you have questions, or want to get involved,{" "}
+                <a
+                    href={
+                        "mailto:macovidvaccines@gmail.com?subject=site feeedback"
+                    }
+                >
+                    send an email send Olivia email
```
Comment (livgust) [inline src/About.js:167]: I like "send an email"

File: src/About.js [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+                </a>
+                .
+            </p>
+
+            {/*            https://www.cnn.com/videos/health/2021/02/08/software-developer-builds-simple-massachusetts-covid-19-vaccine-website-olivia-adams-intv-newday-vpx.cnn*/}
```
Comment (livgust) [inline src/About.js:172]: Mind taking this out?

**Review by livgust — CHANGES_REQUESTED** [review]
Thanks for this, it looks good! 2 tiny nits.

**Review by livgust — APPROVED** [review]

**Pull request merged**